packages/rx-effects/src/declareStore.ts

```diff
@@ -0,0 +1,148 @@
+import {
+  createStore,
+  createStoreUpdates,
+  StateMutation,
+  StateUpdates,
+  Store,
+  StoreWithUpdates,
+} from './store';
+
+interface StoreOptions<State> {
```

Comment:
Please use the original `StoreOptions` from `store.ts` to allow changing `name` for a specific store instance.